src/Support/Http/Controllers/GraphQLController.php

```diff
@@ -88,12 +90,21 @@ public function query(Request $request): Response
      */
     protected function execute(Request $request, GraphQLContext $context): array
     {
+        $operations = json_decode($request->input('operations'));
+
+        $query = optional($operations)->query ?: $request->input('query', '');
```

Comment:
Is optional even necessary here?

Reply:
Yes. Otherwise an ErrorException will be thrown when using a traditional POST-request:
`Trying to get property 'query' of non-object`

However, I have wrapped `json_decode ` in `optional()` instead of using it for each parameter.